<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="suresh-pc" name="Topic9_Timeout_Testng.Example1_Timeout" tests="1" failures="0" timestamp="29 Jun 2016 11:25:59 GMT" time="2.028" errors="1">
  <testcase name="testcase1" time="2.028" classname="Topic9_Timeout_Testng.Example1_Timeout">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.testcase1() didn&#039;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.testcase1() didn't finish within the time-out 2000
	at java.lang.ThreadGroup.remove(Unknown Source)
	at java.lang.ThreadGroup.threadTerminated(Unknown Source)
	at java.lang.Thread.exit(Unknown Source)
]]>
    </error>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- Topic9_Timeout_Testng.Example1_Timeout -->
